packages/arb-validator/test/challenge_test.go

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright 2019, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	jsonenc "encoding/json"
+	"fmt"
+	"github.com/offchainlabs/arbitrum/packages/arb-validator/bridge"
+	"io/ioutil"
+	"log"
+	"math"
+	"math/big"
+	brand "math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/offchainlabs/arbitrum/packages/arb-validator/channel"
+
+	"github.com/offchainlabs/arbitrum/packages/arb-validator/ethbridge"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/offchainlabs/arbitrum/packages/arb-util/evm"
+	"github.com/offchainlabs/arbitrum/packages/arb-util/value"
+	"github.com/offchainlabs/arbitrum/packages/arb-validator/ethvalidator"
+	"github.com/offchainlabs/arbitrum/packages/arb-validator/loader"
+	"github.com/offchainlabs/arbitrum/packages/arb-validator/valmessage"
+)
+
+func TestChallenge(t *testing.T) {
+
+	bridge_eth_addresses := "../../arb-provider-go/test/bridge_eth_addresses.json"
+	contract := "../../arb-provider-go/test/contract.ao"
```

Comment:
arb-validator shouldn't have a dependence on arb-provider-go. For now it should just have a separate copy of `bridge_eth_addresses.json` and `contract.ao`